### PR TITLE
Synchronous Chinese translation

### PR DESCRIPTION
**Pull request by LiHaiP**

### CODE REVIEW

File: po/zh_CN.po [hunk 1]
```diff
@@ -592,14 +592,38 @@ msgstr ""
...
+
+#: lib/pengine/common.c:201
+msgid "Whether to cancel recurring actions removed from the configuration"
+msgstr "是否取消删除配置的重复操作"
+
+#: lib/pengine/common.c:207
+msgid ""
+"*** Deprecated *** Whether to remove stopped resources from the executor"
+msgstr ""
+"***不推荐***是否删除资源管理器中已停止的资源"
```
Comment (gao-yan) [inline po/zh_CN.po:619]: Let's be more precise like "是否从 pacemaker-execd 守护进程中清除已停止的资源".

File: po/zh_CN.po [hunk 2]
```diff
@@ -592,14 +592,38 @@ msgstr ""
...
 msgid "Whether watchdog integration is enabled"
 msgstr "是否启用watchdog集成设置"
 
+#: lib/pengine/common.c:189
+msgid "Whether the cluster should stop all active resources"
+msgstr "群集是否应停止所有运行的资源"
+
+#: lib/pengine/common.c:195
+msgid "Whether to stop resources that were removed from the configuration"
+msgstr "是否停止删除配置的资源"
```
Comment (gao-yan) [inline po/zh_CN.po:609]: Or should be more clear to go with "配置已被删除的资源"?

File: po/zh_CN.po [hunk 3]
```diff
@@ -592,14 +592,38 @@ msgstr ""
...
+
+#: lib/pengine/common.c:201
+msgid "Whether to cancel recurring actions removed from the configuration"
+msgstr "是否取消删除配置的重复操作"
```
Comment (gao-yan) [inline po/zh_CN.po:613]: Similarly.

**Review by gao-yan — CHANGES_REQUESTED** [review]

**Comment by kgaillot** [comment]
ok to test

File: po/zh_CN.po [hunk 4]
```diff
@@ -664,6 +664,30 @@ msgstr ""
 msgid "Whether watchdog integration is enabled"
 msgstr "是否启用watchdog集成设置"
 
+#: lib/pengine/common.c:189
+msgid "Whether the cluster should stop all active resources"
+msgstr "群集是否应停止所有运行的资源"
+
+#: lib/pengine/common.c:195
+msgid "Whether to stop resources that were removed from the configuration"
+msgstr "配置已被删除的资源"
```
Comment (gao-yan) [inline po/zh_CN.po:673]: Of course to be complete, it should be like "是否停止配置已被删除的资源".

File: po/zh_CN.po (hunk 3 above)
Comment (gao-yan) [reply po/zh_CN.po:677]: So this could be "是否取消配置已被删除的的重复操作".

**Review by gao-yan — CHANGES_REQUESTED** [review]

**Comment by gao-yan** [comment]
We could avoid such a "Merge" commit, for example with `git reset` and `amend` commands on the previously existing commit. And `git push --force` will update the PR.

**Comment by gao-yan** [comment]
Thanks. It seems that you need to rebase your commit onto the main HEAD though.

**Pull request closed without merging**